kwok/main.go

```diff
@@ -34,12 +33,11 @@ func main() {
 
 	cloudProvider := kwok.NewCloudProvider(ctx, op.GetClient(), instanceTypes)
 	op.
-		WithWebhooks(ctx, webhooks.NewWebhooks()...).
 		WithControllers(ctx, controllers.NewControllers(
 			op.Manager,
 			op.Clock,
 			op.GetClient(),
 			op.EventRecorder,
 			cloudProvider,
-		)...).Start(ctx, cloudProvider)
```

Comment:
Note: This is another signal that this change is breaking for this upstream library